<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>269</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnCue04">
       <property name="text">
        <string>04</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue05">
       <property name="text">
        <string>05</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue06">
       <property name="text">
        <string>06</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>269</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="btnCue13">
       <property name="text">
        <string>13</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue14">
       <property name="text">
        <string>14</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue15">
       <property name="text">
        <string>15</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>269</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnCue07">
       <property name="text">
        <string>07</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue08">
       <property name="text">
        <string>08</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue09">
       <property name="text">
        <string>09</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>269</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="btnCue16">
       <property name="text">
        <string>16</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue17">
       <property name="text">
        <string>17</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue18">
       <property name="text">
        <string>18</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>269</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btnCue10">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue11">
       <property name="text">
        <string>11</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue12">
       <property name="text">
        <string>12</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>269</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCue01">
       <property name="text">
        <string>01</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue02">
       <property name="text">
        <string>02</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCue03">
       <property name="text">
        <string>03</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListView" name="listViewSlats">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>141</width>
      <height>361</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSerial">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditLog">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>701</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSerial">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Again</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Control Grp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelControl">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPin">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSignalStrength">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Signal Str</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetCues">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Slat Cues</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetBoard">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Board</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetAll">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFireAll">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartScript">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>500</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Script Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditScript">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>531</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
